aws-lambda-scorer/lambda-template/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     implementation group: 'com.amazonaws', name: 'aws-lambda-java-events'
     implementation group: 'com.amazonaws', name: 'aws-java-sdk-s3'
     implementation group: 'com.google.code.gson', name: 'gson'
+    implementation group: 'com.google.guava', name: 'guava'
```

Comment:
@mmalohlava  is added as a dependency. 

However, I like using the runtimeClasspath as a better solution. (this line is not necessary using the runtimeClasspath to build the zip)

Reply:
👍 agree, runtimeClasspath is safer - in case we are missing/will be missing any other dependency in the future. But we need to test it on actual Lambda - pls prepare a ticket in `h2oai` with upgrade of the deployment templates (and cc @anuvaidya to validate Lambda deployment)

Reply:
agree with Michal here.